docs/tutorials/morpheusvm/morpheusvm.md

```diff
@@ -562,12 +576,24 @@ func (*StateManager) AddBalance(
 }
 ```
 
-Finally, we need to implement `SponsorStateKeys`. This function may be confusing,
-but it's actually incredibly simple. HyperSDK uses the `chain.Auth` type to handle
-signatures and treats account abstraction as a first class citizen. This means that
-`chain.Auth` returns both an `Actor`, the subject performing an action like a
-`Transfer`, and `Sponsor`, which returns the address responsible for paying fees
-of a transction.
+Finally, we need to implement `SponsorStateKeys`.
```

Comment:
Make sure that we provide an explanation of `state.Keys` when introducing it for the first time